### PR TITLE
fix(client): make alt text on images appear in dark mode

### LINKED ISSUE #9130: alt text of images not visible in dark theme (a11y)
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Performance/Animation_performance_and_frame_rate

### What specific section or headline is this issue about?

it's about the styling of img elements in general

### What information was incorrect, unhelpful, or incomplete?

I'm not quite sure where this issue belongs, but I've already mentioned it in mdn/content#27241 (it was said to file a separate issue)

The styling of `img` elements sets a white background (index.scss).
`.main-page-content img { background: #fff; }`

The problem is that the alt text of images is not visible due to the background color set (see screenshots below).

I suggest to set the background-color to `transparent` for the dark theme (or `unset` it), or to set the text `color` to a (valid contrast ratio) dark color (e.g. the primary text color in light mode - but not with the custom property, because the custom property is also changed in the dark theme...).

### What did you expect to see?

The alt text to be readable using the dark theme

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

**light theme**

light-theme-img

---

**dark theme**

dark-theme-img

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/performance/animation_performance_and_frame_rate`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Performance/Animation_performance_and_frame_rate
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/performance/animation_performance_and_frame_rate/index.md
* Last commit: https://github.com/mdn/content/commit/6bff95b8adb7ecb18660320721433525a26360d1
* Document last modified: 2023-02-24T06:17:52.000Z

</details>

### PR DESCRIPTION
**Pull request by CBID2**
## Summary

Fixes #9130 
### Problem

The alt text of images is not visible due to the way the background color is set
### Solution

I made set the background color to `transparent` for the dark theme. This will make the alt text appear in dark mode. 
---

### CODE REVIEW

**Comment by bsmth** [comment]
Thanks a lot for the changes! I tried this fix out myself with an image with a wrong src:

<details>
image

image
</details>

Looks good, in my opinion. I don't know if it affects any other places where we have images (pngs?) with a transparent background. It would also be great for Yari dev to have a look, also.

File: client/src/document/index.scss [hunk 1]
```diff
@@ -106,7 +106,7 @@
   }
 
   img {
-    background: #fff;
+    background: transparent;
```
Comment (caugner) [inline client/src/document/index.scss:109]: Let's use `--background-primary` instead to be safe:

```suggestion
    background: var(--background-primary);
```

**Review by caugner — CHANGES_REQUESTED** [review]
Thank you, `transparent` works, but we should better use the `--background-primary` CSS variable instead to keep the behavior similar, which is white with the light theme and 90% black with the dark theme.

**Comment by CBID2** [comment]
> Thank you, `transparent` works, but we should better use the `--background-primary` CSS variable instead to keep the behavior similar, which is white with the light theme and 90% black with the dark theme.

Done @caugner! :)

**Review by caugner — APPROVED** [review]
Thank you! 🙌

**Pull request merged**